instrumentation/cassandra/README.md

```diff
@@ -0,0 +1,52 @@
+# brave-instrumentation-cassandra
+This contains a tracing implementation for [Cassandra](https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/tracing/Tracing.java).
+
+`Tracing` extracts trace state from the custom payload of incoming
+requests. How long each request takes, along with relevant tags like the
+session ID are reported to Zipkin.
+
+## Client side requirements
+Clients must enable tracing. If they want to continue an existing trace,
+they must add propagation fields to the custom payload of a statement.
+
+```java
+// minimally, you need to prepare a statement and enable tracing
+preparedStatement.enableTracing();
+
+// By default, B3 style is used, so instrumented clients do something like this
+Map<String, ByteBuffer> payload = new LinkedHashMap<>();
+payload.set("X-B3-TraceId", byteBuffer("463ac35c9f6413ad"));
```

Comment:
note that brave allows you to change the propagation format. I intentionally decided to not go with the finagle binary encoding as most people don't understand what that is. B3 is reasonably well documented, so a better default IMHO.